src/neon/neon_butterflies.rs

```diff
@@ -112,9 +142,29 @@ macro_rules! boilerplate_fft_neon_f64_butterfly {
                 &self,
                 input: &mut [Complex<T>],
                 output: &mut [Complex<T>],
+            ) -> Result<(), ()> {
+                array_utils::iter_chunks_zipped_mut(
```

Comment:
Should forward to immutable version

Reply:
Updated recommendation is the same as above, `perform_oop_fft_butterfly_multi` should be the only method here